models2/PostModel.es6.js

```diff
@@ -5,6 +5,14 @@ import replyable from './mixins/replyable';
 
 const T = RedditModel.Types;
 
+const cleanThumbnail = thumbnail => {
+  if (thumbnail && (thumbnail === 'default' || thumbnail === 'self' || thumbnail === 'nsfw')) {
```

Comment:
Make an array with `['default', 'self', 'nsfw']` and `ARRAY.indexOf(thumbnail) !== -1`?